src/compiler/checker.ts

```diff
@@ -13916,14 +13916,10 @@ namespace ts {
 
                 checkDecorators(node);
                 checkSignatureDeclaration(node);
+                // TODO (weswig): Consolidate the below into checkAllCodePathsInNonVoidFunctionReturnOrThrow

```

Comment:
The `checkAllCodePathsInNonVoidFunction` actually doesn't handle that case, because for a normal function it is allowed. Removing the below block literally just removes that error (that a `get` accessor must contain an explicit return value) from every baseline.